src/cloud/index.js

```diff
@@ -199,17 +200,40 @@ export async function deploy(options) {
     }
     const enteredTag = await getTagPrompt();
     await warn(environment);
+    const serviceNames = services.map(([service, subservice]) => {
+      let serviceName = service;
+      if (subservice) serviceName += ` / ${subservice}`;
+      return serviceName;
+    });
+    try {
+      slackStartedDeploy(environment, config, serviceNames);
+    } catch (e) {
+      console.info(red('Failed to post to Slack'));
```

Comment:
perhaps do console.error(e) as well

Reply:
is this important?

Reply:
if fails its nice to know why it failed no ... but its not important just a comment, as you see i approve the PR

Reply:
oh I get it now... I meant is it important that it's `console.error`?

Reply:
in terms of formatting not ... 
had to look this up what the diff is 

console.error() writes to stderr, whereas console.log() writes to stdout as described in the doc.

https://nodejs.org/api/console.html#console_console_error_data_args

Reply:
right but does that matter to us here?

Reply:
ohh sure ... dont care just care that we output the error

Reply:
For starters I just wanted to have it fail silently, not bugging the user. I later opted to at least show a generic Slack error. 

I agree with Kaare that it would be better/nicer to write to stderr with `console.error`.